src/Client.php

```diff
@@ -44,7 +44,7 @@ public function get($idWithCommand)
         return $content;
     }
 
-    public function post($content, $filename)
+    public function post($content, $filename, $retries = null)
```

Comment:
Please don't change public interface of this library. Let it just do 12 retries with 10 seconds delay. This will give us 2 minutes possible server outage.